info_terminal/infremen/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(infremen)
+
+find_package(catkin REQUIRED COMPONENTS  mongodb_store roscpp actionlib_msgs actionlib) 
+
+set ( CMAKE_CXX_FLAGS "-Ofast -march=native -mno-avx -ggdb")
+
+catkin_package(CATKIN_DEPENDS roscpp message_runtime actionlib_msgs strands_executive_msgs std_msgs strands_navigation_msgs mongodb_store_msgs geometry_msgs)
```

Comment:
This should only contain packages that are also listed in find_package and the package.xml
* `message_runtime` is not necessary
* `actionlib_msgs` is not necessary
* `std_srvs` is missing